src/CustomFunction.ts

```diff
@@ -13,7 +13,7 @@ import {
   FunctionExecutedEvent,
 } from './types';
 import processMiddleware from './middleware/process';
-import { CustomFunctionInitializationError } from './errors';
+import { CustomFunctionCompleteFailError, CustomFunctionCompleteSuccessError, CustomFunctionInitializationError } from './errors';
```

Comment:
We might want to just use the `CustomFunctionInitializationError` error here to not confuse initialization and runtime errors 🤔